dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1426,6 +1528,431 @@ async fn cmd_db_snapshot_info(
     Ok(())
 }
 
+/// List all region replacement requests
+async fn cmd_db_region_replacement_list(
+    _opctx: &OpContext,
+    datastore: &DataStore,
+    fetch_opts: &DbFetchOptions,
+    args: &RegionReplacementListArgs,
+) -> Result<(), anyhow::Error> {
+    let ctx = || "listing region replacement requests".to_string();
+    let limit = fetch_opts.fetch_limit;
+
+    let requests: Vec<RegionReplacement> = datastore
+        .pool_connection_for_tests()
```

Comment:
`pool_connection_authorized` is a private method and can't be accessed from omdb. Short of moving these into `DataStore` I'm not sure what to do here except that, so that's done in a155af913, except when the fetch opts limit needs to be honoured.